docs/gettinginvolved/infrastructure/release-schedule-versioning.rst

```diff
@@ -74,21 +83,23 @@ Examples
 
 For example:
 
-+---------------+-------------------------------------------------+
-| ``0.3.0``     | 0.3.0 released                                  |
-+---------------+-------------------------------------------------+
-| ``0.3.1dev1`` | Weekly releases of 0.3.1                        |
-+---------------+-------------------------------------------------+
-| ``0.3.1``     | Micro marketing release                         |
-+---------------+-------------------------------------------------+
-| ``0.3.2dev1`` | Weekly release                                  |
-+---------------+-------------------------------------------------+
-| ``0.3.2dev2`` | Weekly release                                  |
-+---------------+-------------------------------------------------+
-| ``0.4.0pre1`` | Pre-release of 0.4.0                            |
-+---------------+-------------------------------------------------+
-| ``0.4.0``     | Major marketing release                         |
-+---------------+-------------------------------------------------+
++----------------+-------------------------------------------------+
+| ``0.3.0``      | 0.3.0 released                                  |
++----------------+-------------------------------------------------+
+| ``0.3.1dev1``  | Weekly releases of 0.3.1                        |
++----------------+-------------------------------------------------+
+| ``0.3.1``      | Micro marketing release                         |
++----------------+-------------------------------------------------+
+| ``0.3.1+doc1`` | Documentation release of 0.3.1                  |
```

Comment:
`+doc1` is missing the `.` from `+doc.X`. On that note, how come this is the only one with a period?